src/naksu/constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+const (
+  // LowDiskLimit sets the warning level of low disk
+	LowDiskLimit = 50 * 1024 * 1024 // 50 Gb
+
+  // AbittiVagrantURL is the URL for the Abittti Vagrantfile
+  AbittiVagrantURL = "http://static.abitti.fi/usbimg/qa/vagrant/Vagrantfile"
+
+  // URLTest is a testing URL for network connectivity (network.CheckIfNetworkAvailable).
+	// Point this to something ultra-stable
+  URLTest = "http://static.abitti.fi/usbimg/qa/latest.txt"
+
+  // URLTestTimeout is the timeout in seconds for the test above
+  URLTestTimeout = 4
```

Comment:
This constants package is great! Would including the cache timeout constant here be appropriate?

Reply:
+1

Reply:
:+1: